### ISSUE TITLE
AspNetCore: Specify multiple features with "feature" tag helper

### ISSUE THREAD

**Issue by shibayan** [Azure/AppConfiguration#63]
I want it to have the same functionality as the `environment` tag helper built-in ASP.NET Core.

https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/built-in/environment-tag-helper?view=aspnetcore-2.2
https://github.com/aspnet/AspNetCore/blob/master/src/Mvc/Mvc.TagHelpers/src/EnvironmentTagHelper.cs

The `name` attribute is not listed in intellisense, so please fix it as well.

**Comment by jimmyca15** [comment]
Thanks @shibayan, it's a great suggestion! We can include this in our next release.

**Comment by jimmyca15** [comment]
The environment tag didn't cover all the possible use cases of the feature tag helper so our design will be a bit different. We need to be able to

1. Show if **all** specified features are enabled
2. Show if **any** specified features are enabled
3. Hide if **all** specified features are disabled
4. Hide if **any** specified features are disabled

So we will be looking at a design like this where **all complexity is used**:

```
<feature Name="MyFeature1,MyFeature2" Requirement="All">
  <p>My feature gated content</p>
</feature>
<feature  Name="MyFeature1,MyFeature2" Requirement="All" Negate="true">
  <p>One or all of the required features aren't enabled.
</feature>
```

Of course the **simple/common use case** with just one feature will look like this

```
<feature Name="MyFeature1">
  <p>My feature gated content</p>
</feature>
<feature  Name="MyFeature1" Negate="true">
  <p>One or all of the required features aren't enabled.
</feature>
```

**Comment by shibayan** [comment]
It looks good to me. It would be enough for me to make the tag helper implementation conform to the ASP.NET Core standard.
Please use properties to make the function understandable by intellisense :)

**Comment by jimmyca15** [comment]
Resolved in version [1.0.0-preview-009000001-1251](https://www.nuget.org/packages/Microsoft.FeatureManagement.AspNetCore/1.0.0-preview-009000001-1251)

[Release Notes](https://github.com/Azure/AppConfiguration/blob/master/releaseNotes/Microsoft.Featuremanagement.md#100-preview-009000001-1251---june-20-2019)
